src/web_scrapper.py

```diff
@@ -0,0 +1,108 @@
+import os
+import requests
+import cv2
+import numpy as np
+from googleapiclient.discovery import build
```

Comment:
my pycharm cannot install googleapiclient. are there some special requirements for that, maybe you had similar problem?

Reply:
I added library to requirements. It will be easier to install. It is actually google-api-python-client.